packages/appium/docs/en/index.md

```diff
@@ -1,61 +1,36 @@
 ---
-title: Appium Documentation
----
-
-# Appium Documentation
+hide:
+  - navigation
+  - toc
 
+title: Welcome
+---
+<style>
+  .md-typeset h1 {
+    display: none;
+  }
+</style>
 <div style="text-align: center">
   <img src="assets/images/appium-logo-horiz.png" style="max-width: 400px;" />
 </div>
 
 Welcome to the Appium documentation! Appium is an open-source project and ecosystem of related
 software, designed to facilitate UI automation of many app platforms, including mobile (iOS,
 Android, Tizen), browser (Chrome, Firefox, Safari), desktop (macOS, Windows), TV (Roku, tvOS,
-Android TV, Samsung), and more.
-
-If you're new to Appium, it's recommended that you start off with the [Introduction](intro/), then
-move on to the [Quickstart](quickstart/). And you can always find Appium's code on GitHub at
-[appium/appium](https://github.com/appium/appium).
-
-## What is Appium for?
```

Comment:
Was this prose section moved somewhere else? Or did you just think it wasn't useful?

Reply:
The first paragraph about UI automation is fairly generic. I don't really expect people to stumble across this documentation without already knowing what UI automation is, and that Appium can be used for that purpose. And the preceding welcome paragraph basically already explains this.
The second paragraph already has a more in-depth version on the 'Intro to Appium'/'How Does Appium Work?' page. I think the basic info I've left in the 'Appium in a Nutshell' page will be sufficient.